dnf-behave-tests/features/environment.py

```diff
@@ -193,6 +214,13 @@ def ftpd_context(context):
     context.ftpd.shutdown()
 
 
+@fixture
+def osrelease(context):
+    context.osrelease = OSRelease('/usr/lib/os-release')
```

Comment:
I'm afraid that with this fixture the tests (as a whole) are not non-destructive any more. When something fails, you can easily end with corrupted /usr/lib/os-release file. I think we should at least identify those destructive test (e.g. by tagging them as @destructive) and add some mechanism to prevent running them (if not explicitly asked to) on localhost outside of container.

Reply:
Good point, I somehow neglected this. I'll take a look, thanks.